### ISSUE TITLE
Getting the wrong error on a declaration in the wrong context

### ISSUE THREAD

**Issue by JsonFreeman** [Microsoft/TypeScript-Sublime-Plugin#43]
``` ts
interface Name {
    nameId: string;
}
interface Data {
    contents: {};
}
var map;
static C; // error on this line
```

The compiler gives the error:
"Expression expected"

Sublime gives the error:
"Declaration or statement expected"
